### PR TITLE
Create ISSUE_TEMPLATE.md

### PR DESCRIPTION
**Pull request by liiight**
Issue template. We'll make life easier for all involved (but most importantly us :laughing: )

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+### Actual behaviour:
+
+### Steps to reproduce:
+
+### Added information:
+
+Please verify that the following cheboxes are checked before submitting your issue:
+
+- [ ] Link to a paste service ([pastebin](http://pastebin.com/) for example) of relevant config (preferabbly full config). Remeber to redact any personal information.
```
Comment (cvium) [inline ISSUE_TEMPLATE.md]: typos galore

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+### Expected behaviour:
+
+### Actual behaviour:
+
+### Steps to reproduce:
+
+### Added information:
+
+Please verify that the following cheboxes are checked before submitting your issue:
```
Comment (cvium) [inline ISSUE_TEMPLATE.md]: typos

File: ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+### Expected behaviour:
+
+### Actual behaviour:
+
+### Steps to reproduce:
+
+### Added information:
```
Comment (cvium) [inline ISSUE_TEMPLATE.md]: Maybe have subheaders here for config and logs.

File: ISSUE_TEMPLATE.md (hunk 2 above)
Comment (liiight) [inline ISSUE_TEMPLATE.md]: I thought i'd give it a more human touch.

**Pull request merged**